packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1891,11 +1892,13 @@ function mountLazyComponent(
     }
   }
 
+  const loggedComponent = Component === REACT_FRAGMENT_TYPE ? '<Fragment>' : Component;
```

Comment:
We can use `getComponentNameFromType` from `shared/getComponentNameFromType` here instead. That way we also get a better name for all the other types you're not supposed to use.
```suggestion
  const loggedComponent = getComponentNameFromType(Component) || String(Component);
```

Reply:
Forgot that we use `+ ''` for perf so `|| (Component + '')` it is.